GooglePlacesAutocomplete.js

```diff
@@ -348,7 +354,7 @@ export default class GooglePlacesAutocomplete extends Component {
   }
 
   _getPredefinedPlace = (rowData) => {
-    if (rowData.isPredefinedPlace !== true) {
+    if (rowData.isPredefinedPlace !== true || rowData.description == null) {
```

Comment:
What affect does this have?

Reply:
This change prevents the method from returning the wrong predefined place.

This method is called in `_onPress` when the user selects a row that references to a predefined place; it attempts to match the selected row back to the original predefined place object. However, this method may return the wrong predefined place if multiple predefined places have no description, because `this.props.predefinedPlaces[i].description === rowData.description` returns true when both sides are null or undefined.

Reply:
Interesting. Thanks for explaining the reasoning behind this. It was hard to tell what exactly it was doing by looking at the code and some quick testing on my end.